bitnami/keycloak/values.yaml

```diff
@@ -70,7 +70,7 @@ diagnosticMode:
 image:
   registry: docker.io
   repository: bitnami/keycloak
-  tag: 16.1.1-debian-10-r85
+  tag: 17.0.1-debian-10-r0
```

Comment:
Not an issue, but we can leave this as it is. The internal CI will update these tags regardless.